<launch>
    <node pkg="mask_rcnn_ros" name="mask_rcnn_node" type="mask_rcnn_node.py" output="screen"/>
    <node pkg="mask_rcnn_ros" name="mask_region_growing_node" type="mask_region_growing_node" output="screen"/>
</launch>
